### PR TITLE
Add how-to-mint-erc721-using-openzeppelin (Round 2)

### PR DESCRIPTION
**Pull request by therealharpaljadeja**

### CODE REVIEW

**Comment by jpop32** [comment]
Please add the images used into the PR and link them relative to root of tutorial (currently they are linked out to your repo). Also, please rename the image files so they have a common and unique prefix that can identify them as belonging to this tutorial (GitBook tool we use puts all the images in a single direcory). For example
minting-erc721-00-name-if-first-image.png, 
minting-erc721-01-name-if-second-image.png,
...

**Comment by jpop32** [comment]
Please add a conclusion / summary part to round out the tutorial. The ending is kinda abrupt.

**Comment by jpop32** [comment]
Please remove the numbering of the steps and format the instructions so they flow naturally: use sentences and paragraphs instead of a list of steps.

**Comment by therealharpaljadeja** [comment]
Hi @jpop32, I have made the changes as requested please have a look and let me know if there are any more. Thanks 🙂

**Comment by jpop32** [comment]
Thanks, that's great. Unfortunately, you cannot manually add files into the .gitbook directory, that causes problems with syncing. Can you just place the images in the /images directory in your tutorial directory? Apologies, I could have been more clear.

**Comment by therealharpaljadeja** [comment]
Thanks, @jpop32  for the review I have moved the images from .gitbook to images directory in my tutorial directory. Let me know if there are any more changes thanks 🙂.

File: build/tutorials/tutorials-contest/how-to-mint-erc721-using-openzeppelin/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
+# How to mint your own ERC721 on Avalanche using Open Zeppelin
+
+### This tutorial teaches you to deploy your own ERC721 token on Avalanche FUJI C-chain testnet.
```
Comment (cgcardona) [inline build/tutorials/tutorials-contest/how-to-mint-erc721-using-openzeppelin/tutorial.md:3]: [Heading levels should only increment by one level at a time](https://github.com/DavidAnson/markdownlint/blob/v0.23.1/doc/Rules.md#md001)

File: build/tutorials/tutorials-contest/how-to-mint-erc721-using-openzeppelin/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+### The steps to deploy on the mainnet are identical and the differences are mentioned in the tutorial.
+
+## Contents
```
Comment (cgcardona) [inline build/tutorials/tutorials-contest/how-to-mint-erc721-using-openzeppelin/tutorial.md:7]: [Headings should be surrounded by blank lines](https://github.com/DavidAnson/markdownlint/blob/v0.23.1/doc/Rules.md#md022)

File: build/tutorials/tutorials-contest/how-to-mint-erc721-using-openzeppelin/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+- [Getting images ready to be uploaded to decentralized storage.](#getting-images-ready-to-be-uploaded-to-decentralized-storage) (IPFS in this tutorial)
```
Comment (cgcardona) [inline build/tutorials/tutorials-contest/how-to-mint-erc721-using-openzeppelin/tutorial.md:8]: Replace `-` w/ `*` for list items to match the rest of the docs

File: build/tutorials/tutorials-contest/how-to-mint-erc721-using-openzeppelin/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+- [Getting metadata ready to be uploaded to decentralized storage.](#getting-metadata-ready-to-be-uploaded-to-decentralized-storage) (IPFS in this tutorial)
+- [Writing code in Remix IDE](#writing-code-in-remix-ide)
+- [Compiling and Deploying to Avalanche FUJI C-Chain Testnet.](#compiling-and-deploying-to-avalanche-fuji-c-chain-testnet-)
+- [Check the NFT on the explorer.](#check-the-nft-on-the-explorer)
+
+### <u>Getting images ready to be uploaded to decentralized storage.</u>
```
Comment (cgcardona) [inline build/tutorials/tutorials-contest/how-to-mint-erc721-using-openzeppelin/tutorial.md:14]: Underlined text is reserved for links. Remove `<u>` tag

File: build/tutorials/tutorials-contest/how-to-mint-erc721-using-openzeppelin/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+
+![metadata-file-in-folder](images/minting-erc721-08-metadata-file-in-folder.png)
+
+Upload the folder of metadata to pinata.
+
+![pinata-metadata-folder-uploaded](images/minting-erc721-09-pinata-metadata-folder-uploaded.png)
+
+You can click on the folder name and see the contents on IPFS.
+
+<br>
```
Comment (cgcardona) [inline build/tutorials/tutorials-contest/how-to-mint-erc721-using-openzeppelin/tutorial.md:105]: Remove `<br>` tag

File: build/tutorials/tutorials-contest/how-to-mint-erc721-using-openzeppelin/tutorial.md [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+
+### <u>Conclusion</u>
+
+We have successfully minted an ERC721 on the AVAX blockchain. OpenZeppelin is a doing a pretty great job at providing boilerplate code to get started however there is a lot more things you can do to an NFT. Make the price go up on every mint 🤑 or making the NFT artwork dynamic by serving the images from a server instead of IPFS 😎. You may also use alternate decentralized storage solutions like Arweave.
+
+I recommend diving more deep and understand every function available to override (basically editing it your way). You can learn more about it [here](https://docs.openzeppelin.com/contracts/4.x/erc721) and [here](https://docs.openzeppelin.com/contracts/4.x/api/token/erc721)!
+
+Happy Minting! 🙂
+
```
Comment (cgcardona) [inline build/tutorials/tutorials-contest/how-to-mint-erc721-using-openzeppelin/tutorial.md:221]: Only have 1 blank line at the end of the file.

File: build/tutorials/tutorials-contest/how-to-mint-erc721-using-openzeppelin/tutorial.md [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+https://cchain.explorer.avax-test.network/tokens/0x10075f07b799f9ce7a585e95a2711766b1e248a2/instance/0/token-transfers
+
+![nft-on-explorer](images/minting-erc721-23-nft-on-explorer.png)
+
+The format is as follows.
+
+*Testnet* - 
+https://cchain.explorer.avax-test.network/tokens/{contract-address}/instance/{tokenId}/token-transfers
+
```
Comment (cgcardona) [inline build/tutorials/tutorials-contest/how-to-mint-erc721-using-openzeppelin/tutorial.md:208]: Only have 1 blank line between paragraphs.

**Review by cgcardona — CHANGES_REQUESTED** [review]
Minor changes requested

**Comment by therealharpaljadeja** [comment]
Thanks, @cgcardona for the constructive feedback. I have made the changes please have a look and let me know if any more changes 😅

**Pull request merged**